config/gitleaks.toml

```diff
@@ -2154,11 +2161,18 @@ keywords = [
 [[rules]]
 id = "huggingface-access-token"
 description = "Discovered a Hugging Face Access token, which could lead to unauthorized access to AI models and sensitive data."
-regex = '''(?:^|[\\'"` >=:])(hf_[a-zA-Z]{34})(?:$|[\\'"` <])'''
-entropy = 1
+regex = '''(?:^|[\\'"\x60>=:\s])(hf_[a-zA-Z0-9]{34})(?:$|[\\'"\x60<\s])'''
+entropy = 3
 keywords = [
     "hf_",
 ]
+[rules.Verify]
+httpVerb = "GET"
+url = "https://huggingface.co/api/whoami-v2"
+headers = {headers = "Bearer ${huggingface-access-token}"}
```

Comment:
@rgmz shouldn't this be `headers = { "authorization" :  "Bearer ${huggingface-access-token}"}`

Reply:
Absolutely. I got distracted by a pattern issue and completely missed this.

Reply:
Fixed.